plugin-server/src/config/kafka-topics.ts

```diff
@@ -52,3 +52,9 @@ export const KAFKA_COHORT_MEMBERSHIP_CHANGED = `${prefix}cohort_membership_chang
 // Error tracking topics
 export const KAFKA_ERROR_TRACKING_ISSUE_FINGERPRINT = `${prefix}clickhouse_error_tracking_issue_fingerprint${suffix}`
 export const KAFKA_ERROR_TRACKING_ISSUE_FINGERPRINT_OVERRIDES = `${prefix}clickhouse_error_tracking_issue_fingerprint_overrides${suffix}`
+
+// Logs ingestion topics
+export const KAFKA_LOGS_INGESTION = `${prefix}logs_ingestion${suffix}`
```

Comment:
is it confusing that capture will write to logs_ingestion or is that the standard pattern we have?

in my head we would write to e.g. "logs_captured", which ingestion would read from, and write to "logs_processed" or "logs_ingested" or whatever. but maybe logs_ingestion -> clickhouse_logs is closer to our existing stuff?

Reply:
I'm actually just trying to stay consistent with our existing inconsistent naming. This matches how we do it for events so lets keep it that way (consistently inconsistent is still preferable)